pygmt/tests/baseline/test_colorbar_frame_string_parameters.png.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
+- md5: 7a8bbd25a40214160f00a5c5bf83f69d
+  size: 3764
+  path: test_colorbar_frame_string_parameters.png
```

Comment:
Could you delete this file and see if it fixes the test failure? Maybe it's still comparing the old image?

Reply:
Still no luck, tried deleting both the .dvc file and the image.